charts/tigera-operator/values.yaml

```diff
@@ -81,7 +81,8 @@ tigeraOperator:
   version: master
   registry: quay.io
 calicoctl:
-  image: quay.io/calico/ctl
+  image: ctl
+  registry: quay.io/calico
```

Comment:
Same here - you can see that the registry for `tigera/operator` is `quay.io` - we should follow that definition of what a registry is.

Reply:
Stated this on a different comment but `tigera/operator` follows a different format from monorepo. This follows the current way the monorepo operates. See [defaultRegistries](https://github.com/projectcalico/calico/blob/master/release/pkg/manager/calico/manager.go#L42-L48)
Changing this from `quay.io/calico` with image as `node` to `quay.io` with image as `calico/node` will require changing a lot more beyond the scope of this PR